releases/release-1.24/release-notes/release-notes-draft.json

```
@@ -3078,30 +3786,1041 @@
+    "pr_number": 108561,
+    "kinds": [
+      "feature"
+    ],
+    "sigs": [
+      "storage"
+    ],
+    "feature": true
+  },
+  "108611": {
+    "commit": "dd2596e40d1fbb92fa4825145c4cf8cda1e559fa",
+    "text": "topology translation of in-tree vSphere volume to vSphere CSI.",
+    "markdown": "Topology translation of in-tree vSphere volume to vSphere CSI. ([#108611](https://github.com/kubernetes/kubernetes/pull/108611), [@divyenpatel](https://github.com/divyenpatel)) [SIG Storage]",
+    "author": "divyenpatel",
+    "author_url": "https://github.com/divyenpatel",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/108611",
+    "pr_number": 108611,
+    "kinds": [
+      "bug",
+      "regression"
+    ],
+    "sigs": [
+      "storage"
+    ],
+    "duplicate_kind": true
+  },
+  "108613": {
+    "commit": "0b8a665d508a861713398d0480e4789802c70607",
+    "text": "Fix a bug that out-of-tree plugin is misplaced when using scheduler v1beta3 config",
+    "markdown": "Fix a bug that out-of-tree plugin is misplaced when using scheduler v1beta3 config ([#108613](https://github.com/kubernetes/kubernetes/pull/108613), [@Huang-Wei](https://github.com/Huang-Wei)) [SIG Scheduling and Testing]",
+    "author": "Huang-Wei",
+    "author_url": "https://github.com/Huang-Wei",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/108613",
+    "pr_number": 108613,
+    "areas": [
+      "test"
+    ],
+    "kinds": [
+      "bug"
+    ],
+    "sigs": [
+      "scheduling",
+      "testing"
+    ],
+    "duplicate": true
+  },
+  "108614": {
+    "commit": "b4f7da1ec8e01700924a1ab60cf7d12b5af8e3bb",
+    "text": "Remove support for node-expansion between node-stage and node-publish",
+    "markdown": "Remove support for node-expansion between node-stage and node-publish ([#108614](https://github.com/kubernetes/kubernetes/pull/108614), [@gnufied](https://github.com/gnufied)) [SIG Storage]",
+    "author": "gnufied",
+    "author_url": "https://github.com/gnufied",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/108614",
+    "pr_number": 108614,
+    "kinds": [
+      "cleanup"
+    ],
+    "sigs": [
+      "storage"
+    ]
+  },
+  "108616": {
+    "commit": "b9817c801bcbf9efbea8da1a5b68c83b3ee4ace6",
+    "text": "client.authentication.k8s.io/v1alpha1 ExecCredential has been removed. If you are using a client-go credential plugin that relies on the v1alpha1 API please contact the distributor of your plugin for instructions on how to migrate to the v1 API.",
+    "markdown": "Client.authentication.k8s.io/v1alpha1 ExecCredential has been removed. If you are using a client-go credential plugin that relies on the v1alpha1 API please contact the distributor of your plugin for instructions on how to migrate to the v1 API. ([#108616](https://github.com/kubernetes/kubernetes/pull/108616), [@margocrawf](https://github.com/margocrawf)) [SIG API Machinery and Auth]",
```

not entirely sure about this one but maybe it should be:
```suggestion
    "markdown": "`client.authentication.k8s.io/v1alpha1` ExecCredential has been removed. If you are using a client-go credential plugin that relies on the v1alpha1 API please contact the distributor of your plugin for instructions on how to migrate to the v1 API. ([#108616](https://github.com/kubernetes/kubernetes/pull/108616), [@margocrawf](https://github.com/margocrawf)) [SIG API Machinery and Auth]",
```

---

Yes, correct